code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -461,7 +461,8 @@
 
 /obj/item/storage/box/syndie_kit/chameleon
 	name = "chameleon kit"
-
+////SKYRAT EDIT REMOVAL BEGIN - SYNDICATE MIRROR - (Moved to modular_skyrat/modules/moretraitoritems/code/syndiemirror.dm)
```

Comment:
I dont think you NEED to comment this out;
could you not just use `. = ..()` to get all of these, THEN add your single new one?

Reply:
See the second comment I left lower down